### ISSUE TITLE
Seismic bump dataset is actually seed dataset

### ISSUE THREAD

**Issue by amueller** [openml/openml-data#24]
https://www.openml.org/d/1500
is identical to
https://www.openml.org/d/1499

It should have more features according to the description.

**Comment by amueller** [comment]
There's another dataset, which is 'in preparation' that looks to be correct:
https://www.openml.org/d/40878
also done by @rgmantovani 
That should be activated as a new version and the old version should be deactivated, right? Why is the new version not automatically activated?

**Comment by rgmantovani** [comment]
Yep. My fault when I uploaded for the first time. Later I corrected but it was not activated.

**Comment by amueller** [comment]
Can we activate it and deactivate the old version? We can do that manually now, right?
